packages/web-components/.gitignore

```diff
@@ -24,7 +24,8 @@ Thumbs.db
 UserInterfaceState.xcuserstate
 .env
 
-react-bindings/
```

Comment:
Updating @stencil/react-output-target moved the files to component.* I added an index file to maintain backwards compatibility with react binding imports